src/slider/example/index.ts

```diff
@@ -65,6 +66,7 @@ export class App extends WidgetBase<WidgetProperties> {
 			}),
 			v('h1', {}, ['Disabled slider']),
 			w(Slider, {
+				extraClasses: { root: 's2' },
```

Comment:
as with label PR, should wrap in a div with a functional test class then use that along with `slidercss.root` to target.